core/src/main/java/nl/nn/adapterframework/doc/objects/DigesterXmlHandler.java

```diff
@@ -0,0 +1,44 @@
+package nl.nn.adapterframework.doc.objects;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.StringTokenizer;
+
+public class DigesterXmlHandler extends DefaultHandler {
```

Comment:
Wat doet deze class?

Reply:
Ik heb dit niet geschreven, enkel verplaatst om de IbisDocPipe.class wat overzichtelijker te maken zodat er niet meerder classes zijn in een file.